.jenkins/generate_caches.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Jenkins action to run a benchmark of dynamics.py on Piz Daint
+# 3/11/2021, Tobias Wicky, Vulcan Inc
+
+# Syntax:
+# .jenkins/action/run_standlone.sh <option>
+
+## Arguments:
+# $1: <option> which can be either empty, "profile" or "build_cache"
```

Comment:
optional: May want to update the header of the existing fetch/generate caches scripts, while you're in there.